com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Artistic/Normal/NormalFromHeightNode.cs

```diff
@@ -14,11 +14,15 @@ internal class NormalFromHeightNode : IStandardNode
             functions: new FunctionDescriptor[] {
                 new(
                     "Tangent",
```

Comment:
I see in SG1 generated shader code there is a check for `SHADER_STAGE_RAY_TRACING` keyword and send an error if it is defined. But from the node script, I don't see how the `if` statement gets inserted. Can keep a follow-up item for it in the doc.

Reply:
Oh yes, that's a good point.  This node will only work in the pixel shader and won't work in ray tracing because it's using derivatives.  I'll make sure we have a line item for that in the list of things that still need to be done.